lib/Froxlor/Cron/Dns/DnsBase.php

```diff
@@ -181,22 +181,37 @@ public function reloadDaemon()
 	public function writeDKIMconfigs()
 	{
 		if (Settings::Get('dkim.use_dkim') == '1') {
+			shell_exec('rm -rf ' . escapeshellarg(\Froxlor\FileDir::makeCorrectDir(Settings::Get('dkim.dkim_dkimkeys'))) . 'dkim_*');	// deletes all keys, DF8OE
```

Comment:
generating the config-files for opendkim/amavis should be outsourced as - like we've discussed - it's not necessarily dependend on nameserver usage

Reply:
Shall I create a new file for DKIM (and possibly late DMARC) - DkimBase.php?

Reply:
no, that's not the structure of the Cron\Dns classes...Dkim is not a Base....Dns is a Base or specific implementations like Bind or PowerDns

I'd add a new function in `lib/Froxlor/Cron/System/TasksCron.php` and let this be a new task

Reply:
Maybe even better to have a complete separate cron in `lib/Froxlor/Cron/Mail/` for this as it's dns independend (somehow)

Reply:
I do not know how to solve this...